lib/active_merchant/billing/gateways/blue_snap.rb

```diff
@@ -190,6 +193,19 @@ def add_auth_purchase(doc, money, payment_method, options)
         end
       end
 
+      def add_stored_credentials(doc, options)
+        return unless stored_credential = options[:stored_credential]
+
+        initiator = stored_credential[:initiator]&.upcase
+        initiator = 'SHOPPER' if SHOPPER_INITIATOR.include?(initiator)
+        doc.send('transaction-initiator', initiator) if stored_credential[:initiator]
```

Comment:
Hey @ajawadmirza I see that you have tests for MIT transactions with the initiator set to `merchant`, but the only initiator option indicated here is `SHOPPER` for CIT transactions. Is calling out both in the code unnecessary? Just wondering for my own knowledge. Your tests are passing and everything else looks good so I'll approve and merge in the morning after reading your response.

Reply:
Hi @rachelkirk ,

From the `stored_credential` object we receive three type of initiator, i.e. merchant, customer, cardholder. On the gateway field mapping merchant is passed as it is passed in the stored_credential object but for customer and cardholder it passes with field value `SHOPPER` thats why we pass in merchant as it is and for customer and cardholder we convert them to the valid required value.